core/templates/dev/head/components/forms/schema_editors/schema_based_unicode_editor_directive.html

```diff
@@ -16,6 +16,7 @@
            ng-keypress="onKeypress($event)" placeholder="<[getPlaceholder()]>" apply-validation="validators()"
            style="display: inline;">
   </span>
+  <span>{{displayPopUpMessage()}}</span>
```

Comment:
Here, do `<[textInputHelperMessage]>`. `<[...]>` is our replacement for the more usual `{{...}}` (see the interpolation stuff defined in app.js).